index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Umoru's portfolio</title>
+  <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;500;600;700;800&family=Roboto:wght@300;400;500&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css"
+      integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="style.css" />
+</head>
+<body>
+  <div class="container">
+    <header class="header">
+      <nav class="navbar">
+        <a class="logo" href="#">My Logo</a>
+        <div class="hamburger">
+        <img src="./images/outline_menu.png" alt="">
+        </div>
+      </nav>
+  </header>
+
+  <section class="mheadline" id="portfolio-id">
+    <div class="text-container">
+      <h1>
+        I'm Umoru<br />
+        Glad to see you!
+      </h1>
+      <p>
+        I'm a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don't hestiate to contact
+        me.
```

Comment:
- [ ] Good job updating your web page according to the require changes gave by the previous CR :clap: , However as he says, this text here should stick to the figma design that means , the font , the color should be the same as in the Figma design , kindly make sure you change that. :smile: 
- [ ] Always see requirements as customers need you have to meet :+1: 

| ### Your design |  ### Figma design |
| :---: | :---: |
|<img width="289" alt="219559090-96598344-a028-4b03-810e-46d609b71b30" src="https://user-images.githubusercontent.com/80458199/219603181-0ca2d054-6a60-41d1-85e6-97f30b4fa318.png">|<img width="464" alt="219559114-3c995628-94de-40eb-b431-ce839b65b7d3" src="https://user-images.githubusercontent.com/80458199/219603212-bdcc004b-9677-438c-909a-627a96ec5b39.png">|